src/components/biggive-campaign-card-filter-grid/test/biggive-campaign-card-filter-grid.spec.tsx

```diff
@@ -34,10 +34,18 @@ describe('biggive-campaign-card-filter-grid', () => {
                 <h4 class="space-above-0 space-below-3 text-colour-primary">
                   Filters
                 </h4>
-                <biggive-form-field-select id="categories" placeholder="Category" space-below="2"></biggive-form-field-select>
-                <biggive-form-field-select id="beneficiaries" placeholder="Beneficiary" space-below="2"></biggive-form-field-select>
-                <biggive-form-field-select id="locations" placeholder="Location" space-below="2"></biggive-form-field-select>
-                <biggive-form-field-select id="funding" placeholder="Funding" space-below="2"></biggive-form-field-select>
+                <div class="select-wrapper-1">
```

Comment:
took a while updating the tests after editing the production code. Not really sure these tests are very valuable, we always have to do manual testing anyway.